web/modules/custom/contact_form/config/install/field.field.message.contact_form_submission.field_contact_copy.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.message.field_contact_copy
+    - message.template.contact_form_submission
+id: message.contact_form_submission.field_contact_copy
+field_name: field_contact_copy
+entity_type: message
+bundle: contact_form_submission
+label: 'Send yourself a copy'
+description: 'Send yourself a copy of this message.'
```

Comment:
It's not necessary to have both the label and the description since they both say basically the same thing. Just having the label is sufficient. If you think "Send yourself a copy" is not clear enough we can change the label to "Send yourself a copy of this message".